test/core.ts

```diff
@@ -19,8 +16,29 @@ describe("Functional Tests", () => {
   beforeEach(setupFixture);
   afterEach(cleanupFixture);
 
+  it("validates that missing api event is sent when required api is missing", (done) => {
```

Comment:
Since you changed the naming to 'features', please update the test name/comments as well